nipple.py

```diff
@@ -29,27 +37,30 @@
         import os
         if force_playsound:
             print "install playsound, when forcing it, or use pygame"
-        else: 
+        else:
             print "install either pygame or playsound (e.g. via pip)"
         os.sys.exit(1)
 
 
 def play(sound_file):
-    if use_pygame:
-        try:
-            global current_sound_file
-            pygame.mixer.music.stop()
-            if not sound_file is current_sound_file:
-                current_sound_file = sound_file
-                pygame.mixer.music.load(sound_file)
-            pygame.mixer.music.play()
-        except pygame.error:
-            logger.debug('no file "%s" found' % sound_file)
+    if os.path.isfile(sound_file):
```

Comment:
the `except pygame.error` and `except PlaysoundException` are handling this case, so this shouldn't be necessary

Reply:
Unfortunately it is necessary. The playsound implementation for mac crashes when the file does't exist.